test/unit/app/get-author.js

```diff
@@ -1,4 +1,4 @@
-import GetAuthor from '../../src/app/lib/get-author';
+import GetAuthor from '../../../src/app/lib/get-author';
```

Comment:
The convention is to capitalise the first letter if it is a constructor intended to be used with `new`. Never a function.